packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -575,6 +575,66 @@ export class PipelineEditor extends React.Component<
     });
   }
 
+  async handleOpenPipeline(): Promise<void> {
```

Comment:
I think there was some miscommunication here.  Originally, I was referring to the `convertPipeline()` method name because, at the time, there was no prompt to ask the user if they should convert, and since it was called unconditionally (with no indication a conversion might be performed), the name `convertPipeline()` seemed better suited to be called `openPipeline()` (or even `loadPipeline()`).  

Since then you added the prompt so that `convertPipeline()` is ONLY called in cases where conversion is appropriate and desired.  And, in the current case, the name of the calling method doesn't need to be changed - so if you prefer `handleLoadPipeline()`, that makes perfect sense.  At this point, it's more about whether you want to change it back or not.  I just wanted to share that the context of my original comment was invalidated once the prompting logic was added.

Sorry about that.  That's a case where waiting on reviewing is best.

Reply:
No worries, as we also added `handleClosePipeline` i think the `handleOpenPipeline` is ok :) these are no public APIs, so we can always change our minds :)